lib/bolt/applicator.rb

```diff
@@ -43,25 +43,38 @@ def custom_facts_task
       @custom_facts_task ||= begin
         path = File.join(libexec, 'custom_facts.rb')
         file = { 'name' => 'custom_facts.rb', 'path' => path }
-        metadata = { 'supports_noop' => true, 'input_method' => 'stdin' }
+        metadata = { 'supports_noop' => true, 'input_method' => 'stdin',
+                     'implementations' => [
+                       { 'name' => 'custom_facts.rb' },
```

Comment:
Is it necessary to have both entries? From the task spec it seems like not https://github.com/puppetlabs/puppet-specifications/tree/master/tasks#options-2 
```
remote: This implementation is remote. Set remote on specific implementations if the task supports both normal and remote execution.(rev 4)
```

Reply:
```Set remote on specific implementations if the task supports both normal and remote execution.```

Is about a task with a remote implemenation and a local implementation. This is a single implementation used for both.